src/platform/i386/Makefile

```diff
@@ -41,7 +41,8 @@ OBJS += serial.o
 OBJS += timer.o
 OBJS += chal.o
 OBJS += boot_comp.o
-OBJS += console.o
+#OBJS += console.o
```

Comment:
If this is commented out, isn't it going to do nothing when `ENABLE_SERIAL` is defined?  Why is this not compiled now?